### PR TITLE
LPS-198116 Fix Object Definition Node Position

### PR DESCRIPTION
**Pull request by igor-franca**
related [ticket](https://liferay.atlassian.net/browse/LPS-198116)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ import {ObjectDefinitionNode} from '../ObjectDefinitionNode/ObjectDefinitionNode
 import './Diagram.scss';
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:22]: @igor-franca, could you put this css import below all the others?

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx [hunk 2]
```diff
@@ -157,6 +157,19 @@ function DiagramBuilder({
 		};
 
 		API.putObjectFolderByExternalReferenceCode(updatedObjectFolder);
+		const {edges, nodes} = store.getState();
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:160]: @igor-franca, could you put blank lines around this line?

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx [hunk 3]
```diff
@@ -157,6 +157,19 @@ function DiagramBuilder({
 		};
 
 		API.putObjectFolderByExternalReferenceCode(updatedObjectFolder);
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:159]: @igor-franca, please put a `await` statement here

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx (hunk 2 above)
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx:160]: Add a blank line here

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts [hunk 4]
```diff
@@ -119,6 +119,18 @@ export type TAction =
 			};
 			type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE;
 	  }
+	| {
+			payload: {
+				newPosition: {
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts:124]: @igor-franca, I think its better we rename this prop to `newObjectDefinitionNodePosition`, what do you think?

**Review by danvb112 — APPROVED** [review]
Hey @igor-franca, good work on this bug, I have a few suggestions :smile:

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, good work on this bug, I have a few suggestions 😄

**Comment by igor-franca** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Comment by igor-franca** [comment]
ci:test:object

**Pull request closed without merging**